.github/workflows/release.yml

```
@@ -0,0 +1,53 @@
+        
+      - name: Run tests
+        run: npm test
+        
+      - name: Run build
+        run: npm run build
+        
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npx semantic-release
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)